docs/tools/developer-tools/analytics-platforms.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Analytics Platforms
+description: Explore the full history of Stellar network data.
+sidebar_label: Analytics Platforms
+sidebar_position: 65
+---
+
+# Analytics Platforms
+
+There's a wide range of analytics platforms that make exploring Stellar network data easy and fun! The following platforms allow you to easily create aggregates, visualizations, and dashboards to figure out exactly how the Stellar network is working for you.
```

Comment:
one last nit: 
> figure out exactly the story you want to tell, with Stellar data.

Reply:
Done in 9cbe40eb302ec050d9dc6b2df5b5e29b9d0518e1

Reply:
🙏